File: python/fundamentals/merge_arrays.py

if __name__ == "__main__":
    arr1 = [1,2,3]
    arr2 = [4,5,6]

    # there are two ways to merge array
    # 1. using + operator
    print(arr1+arr2)

    # 2. using extend
    arr1.extend(arr2)
    print(arr1)
    pass

File: golang/fundamentals/merge_arrays.go
package main

import "fmt"

func main() {
	var (
		arr1 []int = []int{1, 2, 3}
		arr2 []int = []int{4, 5, 6}
	)

	// here is how to merge these arrays
	merged := append(arr1, arr2...)
	fmt.Println(merged)
}
